assets/xml/objects/object_ghaka.xml

```diff
@@ -1,12 +1,12 @@
 ﻿<Root>
     <File Name="object_ghaka" Segment="6">
-        <Collision Name="object_ghaka_Colheader_0000E8" Offset="0xE8" />
+        <Collision Name="object_ghaka_Colheader_0000E8" Offset="0xE8" /> <!-- Original name is "goron_hakaisi_bgdatainfo" -->
```

Comment:
No `z2_` prefix?

Reply:
Nope, not in this case:
![image](https://github.com/zeldaret/mm/assets/12245827/b0d3f79c-c450-419e-aca7-e7bd8f43e85c)

I compared the .zsis from MM3D to the extracted collision data from MM64 to make sure I named them right. The `Vec3s` arrays from MM64 can actually be found verbatim in MM3D (assuming the collision vertices didn't change, which they rarely do), so it's pretty easy to line them up.

Reply:
I should also note the one without the `z2_` prefix is unused; only the latter one is used in ObjGhaka.

Reply:
One last thing: since we have a collision viewer in Z64Utils (I forgot about this, despite literally being the person to finish Tharo's implementation). I took a look at what these two pieces of collision look like:
z2_goron_hakaisi_bgdatainfo (used):
![image](https://github.com/zeldaret/mm/assets/12245827/c0401c97-e1d2-4656-8f32-3dda8702089a)

goron_hakaisi_bgdatainfo (unused):
![image](https://github.com/zeldaret/mm/assets/12245827/78ea91fa-fcd1-4642-92ba-7fdf32207a13)

So the unused one is smaller and less detailed.